setup.py

```diff
@@ -26,12 +26,12 @@ def _resolve_prefix(prefix, type):
     # issue: https://github.com/vcstools/wstool/issues/81
     osx_system_prefix = '/System/Library/Frameworks/Python.framework/Versions'
     if type == 'man':
-        if prefix == '/usr':
+        if prefix == '':
             return '/usr/share'
         if sys.prefix.startswith(osx_system_prefix):
             return '/usr/local/share'
     elif type == 'bash_comp':
-        if prefix == '/usr':
+        if prefix == '':
```

Comment:
Reading the code, I think that if you ran the `setup.py` with the option `--prefix /usr` it would now be broken. Maybe these conditionals should be like `if prefix == '' or prefix == '/usr':`?